etc/jitsi-meet-desktop.profile

```diff
@@ -0,0 +1,30 @@
+# Firejail profile for jitsi-meet-desktop
+# Description: Jitsi Meet desktop application powered by Electron
+# This file is overwritten after every install/update
+# Persistent local customizations
+include jitsi-meet-desktop.local
+# Persistent global definitions
+include jitsi-meet-desktop.local
+
+noblacklist ${HOME}/.config/Jitsi Meet
+
+include disable-common.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+
+whitelist ${DOWNLOADS}
+mkdir ${HOME}/.config/Jitsi Meet
+whitelist ${HOME}/.config/Jitsi Meet
```

Comment:
Add `include whitelist-common.inc`

what about whitelist-var-common, whitelist-usr-share-common, whitelist-runuser-common?

Reply:
I do not understand the need for this. It seems to run fine without including all these whitelist rules. Wouldn't this just unnecessarily give it the ability to edit the whitelisted files?

Reply:
> give it the ability to edit the whitelisted files?

Some of the paths there are read-only (in whitelist-common or disable-common).

> It seems to run fine without including all these whitelist rules. 

- `whitelist-common.inc`: for you it works, but it may need some file for there on other setup. this is for easier maintenance 
- wvc, wusc, wruc: add these increases the security. Example wvc: w/o it all files under /var are visible. with wvc only the few whitelisted paths are visible.